app/services/create_repository_service.rb

```diff
@@ -1,16 +1,8 @@
-class RepositoryService
+class CreateRepositoryService
```

Comment:
I still don't like this thing too much. Just look on how it is called from `lib/rmt/scc.rb:140`:

```ruby
@repository_service.create_repository(service, repo_item[:url], repo_item)
```

- When the class is called `CreateRepositoryService`, then its instance should be called `create_repository_service`, no different.
- A service has one purpose, ergo one public method. The class name already holds the information what it does, so the method should not repeat it. The method should be called sth like `call` or `run` and that should be consistent throughout the project (which is easy because it is the first service ... and I'm not even sure we really need this concept for this purpose)
- The method interface cries for refactoring. You send in stuff from `repo_item` two times.
- Inside the method is a `# TODO` which either needs to be done now, or needs a comment wtf is happening there. It seems you punch the id into the scc_id field for non-custom repos :fist_oncoming: Why does this `attributes` object have an `id` in the first place? Maybe this can be cleaned up on the calling side and this mangling can be removed from here.